pydarn/io/borealis/borealis.py

```diff
@@ -291,7 +291,7 @@ def __init__(self, filename: str, borealis_data: Union[dict, OrderedDict],
             'site', 'array', or None. If None (default), array write will be 
             attempted first followed by site. 
         hdf5_compression: str
-            String representing compression type. Default zlib.
+            String representing compression type. 
```

Comment:
the default is difficult. For site writing, default should None, but I can't set the default to None for array writing. For array writing, default should be zlib. These are set up within the writer themselves but if you want to change it I left it as a kwarg. Thoughts?

Reply:
Maybe a comment on how you are following on-site standards thus not compressing. Then for array mention the fast reading with `zlib` compression. That way people can understand the difference in why one is not compressed but the other is. If that makes sense.

Reply:
@mts299

Reply:
added a comment

Reply:
Can you add `hdf5_compression` is a kwarg key name and then remove lines 300-302 since self.compression is not used. From there we discussed having a getter method for the compression type which would call on the actual writer that was used.

Reply:
done